<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Tim Kuo</author>
 <class>CustomerDetailWidgetClass</class>
 <widget class="QWidget" name="CustomerDetailWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户明细</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>顾客明细</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="100000,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,0,0,1,1,0,0,0">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelId">
              <property name="text">
               <string>编号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="idLcdNumber">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="numDigits">
               <number>9</number>
              </property>
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="nameLabel">
              <property name="text">
               <string>姓名</string>
              </property>
              <property name="buddy">
               <cstring>nameLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="nameLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="sexLabel">
              <property name="text">
               <string>性别</string>
              </property>
              <property name="buddy">
               <cstring>sexComboBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="sexComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="cellLabel">
              <property name="text">
               <string>手机</string>
              </property>
              <property name="buddy">
               <cstring>cellLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="cellLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maxLength">
               <number>16</number>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="phoneLabel">
              <property name="text">
               <string>座机</string>
              </property>
              <property name="buddy">
               <cstring>phoneLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="phoneLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maxLength">
               <number>16</number>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="levelLabel">
              <property name="text">
               <string>等级</string>
              </property>
              <property name="buddy">
               <cstring>sexComboBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="levelComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="addressLabel">
              <property name="text">
               <string>地址</string>
              </property>
              <property name="buddy">
               <cstring>descPlainTextEdit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="addressPlainTextEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>22</height>
               </size>
              </property>
              <property name="tabChangesFocus">
               <bool>true</bool>
              </property>
              <property name="documentTitle">
               <string/>
              </property>
              <property name="backgroundVisible">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="descLabel">
              <property name="text">
               <string>备注</string>
              </property>
              <property name="buddy">
               <cstring>descPlainTextEdit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="descPlainTextEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>22</height>
               </size>
              </property>
              <property name="tabChangesFocus">
               <bool>true</bool>
              </property>
              <property name="documentTitle">
               <string/>
              </property>
              <property name="backgroundVisible">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="timesLabel">
              <property name="text">
               <string>消费次数</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="timesLcdNumber">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="numDigits">
               <number>8</number>
              </property>
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
              <property name="intValue" stdset="0">
               <number>12345678</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="consumeLabel">
              <property name="text">
               <string>累积金额</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="consumeLcdNumber">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="smallDecimalPoint">
               <bool>false</bool>
              </property>
              <property name="numDigits">
               <number>8</number>
              </property>
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
              <property name="value" stdset="0">
               <double>0.000000000000000</double>
              </property>
              <property name="intValue" stdset="0">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="depositLabel">
              <property name="text">
               <string>帐户余额</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="depositLcdNumber">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="numDigits">
               <number>8</number>
              </property>
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="imageLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>4</horstretch>
              <verstretch>3</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>240</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>240</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>4</width>
              <height>3</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>:/common/business</pixmap>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>会员卡数据</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="cardsTreeView">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="animated">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="picButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>更换图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="submitButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>提交</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="4"/>
 <resources>
  <include location="../Cosmetic/Cosmetic.qrc"/>
  <include location="../Cosmetic/Cosmetic.qrc"/>
  <include location="../Cosmetic/Cosmetic.qrc"/>
  <include location="../Cosmetic/Cosmetic.qrc"/>
 </resources>
 <connections/>
</ui>
